### DISCUSSION TITLE
Detecting Theme in Hyprland

### DISCUSSION THREAD

**Discussion by tdtooke** [fastfetch-cli/fastfetch#1451] [Ideas]
Is it possible to detect themes that were set via gsettings and qt5ct?  The reason I ask is I just recently installed Hyprland and if I use "theme" it will correctly detect the theme I had set on Gnome and Plasma but when I'm on Hyprland those aren't the themes in use.  I set my GTK theme with gsettings and my QT theme with qt5ct and qt6ct.  They aren't the same as they are on the other desktops so when I run fastfetch while on Hyprland what it will say for theme isn't correct.

**Comment by CarterLi** [comment]
https://github.com/fastfetch-cli/fastfetch/discussions/787

**Reply by tdtooke** [reply]
This isn't a bug report or a feature request.  This is only me asking if it's currently possible to detect theming that was set the way I described.  I genuinely do not know.

**Comment by tdtooke** [comment]
Nevermind, I got it.  I install kvantum-qt5 and that fixed the qt issue and for some odd reason the settings.ini files in my gtk directories weren't matching what I had set with gsettings.  I now have a script that runs that makes sure that's never a problem again.  Nothing on your end.  Just Hyprland makes you work for it a little more than Plasma or Gnome.
